syntaxes/mojo.syntax.json

```diff
@@ -280,7 +280,7 @@
           }
         },
         {
-          "match": "\\b(var|let|alias) \\s*([[:alpha:]_]\\w*)\\b",
+          "match": "\\b(var|let|alias|owned|borrowed|inout) \\s*([[:alpha:]_]\\w*)\\b",
```

Comment:
Are you sure this is the right place to put this? This seems like it would make it so that `owned`/`borrowed`/`inout` are new ways to define variables as statements inside a function. e.g.:

```mojo
fn example():
    inout x = 1
```

That doesn't seem right to me -- I thought these were only valid on (what everyone else calls parameter lists, but maybe we are now calling argument lists), and then it would feel like `meta.function.parameters.python` is a more appropriate block for these changes to be in. What do you think?

Reply:
That's a very good point, and I think you're right.

However -- and I apologize that this wasn't clearer in the commit message -- this commit and pull request is something I hope to soon automate, such that changes to our internal Mojo syntax file would automatically be exported to this repository. So, in other words, this change here is essentially just exporting a code change that has already been made elsewhere.

I think let's proceed like this:
1. I'll start a discussion, based on your feedback here, on our internal copy of this exported file.
2. I'll tweak the workflow for this repository, to automate the exporting of changes to this file.

Reply:
Thanks, this sounds good. Let's continue with this PR as-is for now, then.

Reply:
> Might as well update this to the fix that was landed today

Oh, oops! I accidentally merged just now, but will send an update for the next release.